### DISCUSSION TITLE
Get disconnected when uploading repository

### DISCUSSION THREAD

**Discussion by GastaoCruzUoM** [community/community#36058] [New to GitHub]
Using GitHub Desktop to up publish code for the first time (main). It's a lot of code, takes a while to compress and start pushing, but halfway through I get disconnected. Does anyone have any suggestions?

Thanks!

![image](https://user-images.githubusercontent.com/110921393/195670910-c0e2b8c0-5a82-49e7-942e-0fbf97671a1f.png)

**Comment by airtower-luna** [comment]
How much is "a lot of code"? That makes me wonder if you're running into trouble with size limits: https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#about-size-limits-on-github

**Reply by GastaoCruzUoM** [reply]
It's a lot of files (~4k), but I am ignoring large files. Still, perhaps the total repository size is too big? I will try to split the codebase into multiple repositories.

Thanks!
